client/my-sites/stats/navigation-arrows/index.jsx

```diff
@@ -0,0 +1,42 @@
+import { Icon, arrowLeft, arrowRight } from '@wordpress/icons';
+import classNames from 'classnames';
+
+import './style.scss';
+
+function NavigationArrows( {
+	disableNextArrow,
+	disablePreviousArrow,
+	onArrowNext,
+	onArrowPrevious,
+} ) {
+	const handleClickNext = () => {
+		onArrowNext();
+	};
+	const handleClickPrevious = () => {
+		onArrowPrevious();
+	};
+	return (
+		<div className="arrow-navigation">
+			<button
+				className={ classNames( 'arrow-navigation__previous', {
+					'is-disabled': disablePreviousArrow,
+				} ) }
+				onClick={ disablePreviousArrow ? undefined : () => handleClickPrevious() }
```

Comment:
```suggestion
				onClick={ disablePreviousArrow ? undefined : onClickPrevious }
```

Let's bind the entire function to the event so that component consumers can directly access all event callback parameters. As a bonus, this will help us get rid of the intermediate callback functions`handleClickPrevious` and `handleClickNext` 🙂